protobuf_version.bzl

```diff
@@ -1,3 +1,4 @@
 PROTOC_VERSION = "21.7"
 PROTOBUF_JAVA_VERSION = "3.21.7"
 PROTOBUF_PYTHON_VERSION = "4.21.7"
+PROTOBUF_RUBY_VERSION = "3.21.7"
```

Comment:
This should be 21.8 (post-merge with #10788)

Reply:
Fixed